src/Components/Addbook.js

```diff
@@ -1,21 +1,48 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+/* eslint-disable react/prop-types */
```

Comment:
The goal of linters is to help us write clean code. Kindly remove linters disabling here.